tests/browser/test/basic_block_test.js

```diff
@@ -37,14 +34,8 @@ suite('Basic block tests', function (done) {
 
   test('Drag three blocks into the workspace', async function () {
     for (let i = 1; i <= 3; i++) {
-      await dragBlockTypeFromFlyout(
-        browser,
-        'Basic',
-        'test_basic_empty',
-        250,
-        50 * i
-      );
-      chai.assert.equal((await getAllBlocks(browser)).length, i);
+      await dragNthBlockFromFlyout(browser, 'Align', 0, 250, 50 * i);
+      chai.assert.equal(i, (await getAllBlocks(browser)).length);
```

Comment:
It should be the actual vaue and then the expecte value (swap le arguments).

Reply:
Done. My expectations were backwards!